dashboard/src/app/workspaces/workspace-details/workspace-stacks/workspace-stacks.controller.ts

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (c) 2015-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ */
+'use strict';
+import {CheWorkspace} from '../../../../components/api/che-workspace.factory';
+import {ComposeEnvironmentManager} from '../../../../components/api/environment/compose-environment-manager';
+import {CheEnvironmentRegistry} from '../../../../components/api/environment/che-environment-registry.factory';
```

Comment:
@benoitf I guess we could try to add a che namespace. I think about it. Not in this issue. But I want try to do it. WDYT?